src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +13,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
+        User found = userService.findByEmail(email);
+        if (found != null) {
+            return found.getPassword().equals(password);
+        }
```

Comment:
Think about better name for variable with name `found`, please :)
Also you should think about simplifying of `if` condition and its body. I believe you will do the best 👍